index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome Page</title>
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+    <header>
+    
+        <h1>To-Do</h1>
+        <img src="/Image/logo.png" alt="logo">
+        <p>A simple and intellgent to-do list makes it easy to plan your day</p>
+    
+    </header>
+    <footer>
```

Comment:
in CSS
What about making footer diplay as flex:
and then centering all containing elements instead of playing with margin?